src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/BQSRPipelineSpark.java

```diff
@@ -90,13 +94,20 @@ protected void runTool(final JavaSparkContext ctx) {
                     "as getParallelVariants(List) is broken");
         }
         final JavaRDD<GATKRead> initialReads = getReads();
-        final VariantsSparkSource variantsSparkSource = new VariantsSparkSource(ctx);
 
+        // The initial reads have already had the WellformedReadFilter applied to them, which
+        // is all the filtering that ApplyBQSR wants. BQSR itself wants additional filtering
+        // performed, so we do that here.
+        final ReadFilter bqsrReadFilter = BaseRecalibrator.makeBQSRSpecificReadFilters();
```

Comment:
It seems worse to have to pull this out rather than having it in the tool itself.